index.js

```diff
@@ -14,9 +15,13 @@ app.get('/', (request, response) => {
 
 app.get('/leagues', getAllLeagues)
 
-app.get('/leagues/:leagueId/leads', getAllLeads)
+app.get('/leagues/:leagueSlug/leads', getAllLeads)
 
-app.get('/leagues/:leagueId/leads/:slug', getLeadEncountersBySlug)
+app.get('/leagues/:leagueSlug/leads/:slug', getLeadEncountersBySlug)
+
+app.post('/leagues/:leagueSlug/leads', bodyParser.json(), saveNewEncounter)
+
+app.delete('/leagues/:leagueSlug/leads/:slug/:id', deleteEncounter)
```

Comment:
`/:id` is why this route doesn't work, your documentation is wrong but even if it wasn't this is what I get when I look up that lead. How would I know the encounter id?

![image](https://user-images.githubusercontent.com/5775716/83802422-b9b2a380-a678-11ea-881c-de9e68e8b9fd.png)